ui/k-toolbar/main.kytos

```diff
@@ -1,343 +1,431 @@
+              } else {
+                _this.link_options.push({value:value.id, description:value.id});
+              }
+            });
           }
-        self.show();
-      });
-      request.fail(function(data) {
-        let notification = {
-          icon: 'gear',
-          title: 'Bad request',
-          description: data.status + ': ' + data.responseJSON.description
-        };
-        self.$kytos.$emit("setNotification", notification);
-      });
-    },
-    get_topology(){
-      var self = this
-      $.ajax({
-        async: true,
-        dataType: "json",
-        url: this.$kytos_server_api + "kytos/topology/v3",
-
-        success: function(data) {
-          self.switches = data['topology']['switches']
-          self.links = data['topology']['links']
-        }
-      });
-
-    },
-    show() {
-       var content = {
-                      "component": 'kytos-pathfinder-k-info-panel-best_path',
-                      "content": this.paths,
-                      "icon": "compass",
-                      "title": "Best Paths",
-                      "subtitle": "by kytos/pathfinder"
-                     }
-       this.$kytos.$emit("showInfoPanel", content)
-
-   }
-  },
-
-  computed: {
-    metric_options(){
-      var metric_options = {}
-
-      var bandwidth_options = []
-      var reliability_options = []
-      var delay_options = []
-      var utilization_options = []
-      var priority_options = []
-      var ownership_options = []
-      var spf_attribute_options = []
-
-      bandwidth_options.push({value: false, description: 'Bandwidth', selected: true});
-      bandwidth_options.push({value: true, description: 'Bandwidth (Flexible)'});
-      
-      reliability_options.push({value: false, description: 'Reliability', selected: true});
-      reliability_options.push({value: true, description: 'Reliability (Flexible)'});
-
-      delay_options.push({value: false, description: 'Delay', selected: true});
-      delay_options.push({value: true, description: 'Delay (Flexible)'});
-
-      utilization_options.push({value: false, description: 'Utilization', selected: true});
-      utilization_options.push({value: true, description: 'Utilization (Flexible)'});
-
-      priority_options.push({value: false, description: 'Priority', selected: true});
-      priority_options.push({value: true, description: 'Priority (Flexible)'});
-
-      ownership_options.push({value: false, description: 'Ownership', selected: true});
-      ownership_options.push({value: true, description: 'Ownership (Flexible)'});
-
-      metric_options["bandwidth"] = bandwidth_options
-      metric_options["reliability"] = reliability_options
-      metric_options["delay"] = delay_options
-      metric_options["utilization"] = utilization_options
-      metric_options["priority"] = priority_options
-      metric_options["ownership"] = ownership_options
-
-      return metric_options;
+        });
+      },
+      fetch_dpids: function() {
```

Comment:
The name of this method doesn't reflect what it actually does, it's fetching `interfaces` not `dpids`, if you could rename it for clarity that'd be great, including also renaming `dpids` from the data properties too